python-fips/Dockerfile

```diff
@@ -116,11 +116,12 @@ RUN set -eux; \
 	\
 	apt-mark auto '.*' > /dev/null; \
 	apt-mark manual $savedAptMark; \
-	find /usr/local -type f -executable -not \( -name '*tkinter*' \) -exec ldd '{}' ';' \
-		| awk '/=>/ { so = $(NF-1); if (index(so, "/usr/local/") == 1) { next }; gsub("^/(usr/)?", "", so); printf "*%s\n", so }' \
+	find /usr/local -type f -executable -not \( -name '*tkinter*' \) -exec ldd '{}' \; 2>/dev/null \
+		| awk '/=>/ && !/not a dynamic executable/ && $3 ~ /^\// { print $3 }' \
 		| sort -u \
-		| xargs -r dpkg-query --search \
-		| cut -d: -f1 \
+		| xargs -r -I {} sh -c 'dpkg-query -S "{}" 2>/dev/null | cut -d: -f1 || true' \
+		| grep -v "^$" \
+		| grep -v "^diversion" \
```

Comment:
Unless this needs to be expanded for it to work, I'd like to keep this as-is to keep it as close as possible to the upstream python docker image